### ISSUE TITLE
Building of downsampled stack failed

### ISSUE THREAD

**Issue by raacampbell** [alexanderbrown/masiv#1]
Attempted to build downsampled stack for sample `YH297_161224`. The downsampling appeared to proceed just fine but when it finished I got the following error and no downsampled stack was produced.

```
Error using saveTiffStack
Too many input arguments.

Error in masivStack/writeStackToDisk (line 255)
            saveTiffStack(obj.I, tempFileName, 'g', tagList);

Error in selectMasivStack/newButtonClick (line 163)
                a.writeStackToDisk; %Write the downsampled stack to disk
 
Error using uiwait (line 81)
Error while evaluating UIControl Callback
```

**Comment by raacampbell** [comment]
This looks like a namespace clash. I will fix this by starting work on Issue #2 and placing `saveTiffStack` in a package.

**Comment by raacampbell** [comment]
Fixed in ffca49d72b8830ac7da5f760f1ccb9f72fb6d39a
